src/pages/basic_data/app_admin/central_import/CentralImportDuplicationCheckPage.tsx

```diff
@@ -32,16 +33,27 @@ export const CentralImportDuplicationCheckPage = () => {
   const { identifier } = useParams<IdentifierParams>();
   const [duplicateSearchFilters, setDuplicateSearchFilters] = useState(emptyDuplicateSearchFilter);
   const [registrationIdentifier, setRegistrationIdentifier] = useState('');
+  const [showNotApplicableDialog, setShowNotApplicableDialog] = useState(false);
 
-  const importCandidateQuery = useQuery({
+  const importCandidateSearchQuery = useQuery({
     queryKey: ['importCandidateSearch', identifier],
     queryFn: () => fetchImportCandidates(1, 0, `id:"${identifier}"`),
     meta: { errorMessage: t('feedback.error.get_import_candidate') },
   });
-  const importCandidate = importCandidateQuery.data?.hits[0];
+  const importCandidateSearch = importCandidateSearchQuery.data?.hits[0];
```

Comment:
Flisespikking, men...
```suggestion
  const importCandidateSearchResult = importCandidateSearchQuery.data?.hits[0];
```

Reply:
Enig i at det er bedre. Det er tross alt ett bestemt resultat/treff det er snakk om her, ikke bare "et søk"